components/customLink.tsx

```diff
@@ -69,7 +69,7 @@ export const CustomLink: React.FC<CustomLinkProps> = ({
           title={title}
           {...props}
           style={style}
-          prefetch={!isHomePage}
+          {...(isHomePage ? { prefetch: true } : {})}
```

Comment:
I think I prefer the older method of doing this. this is a little difficult to read.
```suggestion
          prefetch={!isHomePage}
```

Reply:
compiler was complaining about `prefetch{true}` because it has been deprecated, therefore I had to change this.
https://nextjs.org/docs/messages/prefetch-true-deprecated